extensions/src/main/java/org/pentaho/platform/plugin/action/Payload.java

```diff
@@ -0,0 +1,73 @@
+/*!
```

Comment:
Instead of creating a new file, could we have `Payload` declared as an inner class of `ActionInvokerSystemListener` ?